### PR TITLE
*: add tidb_auto_analyze_concurrency

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18651

https://github.com/pingcap/docs-cn/pull/18651

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -1199,6 +1199,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 
+### tidb_auto_analyze_concurrency <span class="version-mark">New in v8.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `2`
+- Range: `[1, 2147483647]`
+- This variable is used to set the concurrency of executing the automatic update of statistics.
```
Comment (qiancai) [inline system-variables.md:1210]: 这个变量和 tidb_auto_build_stats_concurrency 的区别是什么呢，描述看起来是一样的

image

Comment (hawkingrei) [reply system-variables.md:1210]: 这个是一个 analyze 任务内部的并发。上面那个是可以并发执行多少 analyze 任务。

File: system-variables.md [hunk 2]
```diff
@@ -1199,6 +1199,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 1 row in set (0.00 sec)
 ```
 
+### tidb_auto_analyze_concurrency <span class="version-mark">New in v8.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `2`
```
Comment (hawkingrei) [inline system-variables.md:1208]:
```suggestion
- Default value: `1`
```

File: system-variables.md [hunk 3]
```diff
@@ -1199,6 +1199,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 
+### tidb_auto_analyze_concurrency <span class="version-mark">New in v8.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `1`
+- Range: `[1, 2147483647]`
+- This variable is used to set the concurrency of executing the automatic update of statistics.
```
Comment (Oreoxmt) [inline system-variables.md:1210]:
```suggestion
- This variable is used to set the concurrency within a signle automatic statistics collection task. Before v8.4.0, the concurrecncy for each automatic statistics collection task is fixed at `1`. You can increase this concurrency to speed up statistics collection tasks based on your cluster's available resources.
```

**Review by songrijie — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -1199,6 +1199,16 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 
+### tidb_auto_analyze_concurrency <span class="version-mark">New in v8.4.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Integer
+- Default value: `1`
+- Range: `[1, 2147483647]`
+- This variable is used to set the concurrency within a signle automatic statistics collection task. Before v8.4.0, the concurrecncy for each automatic statistics collection task is fixed at `1`. You can increase this concurrency to speed up statistics collection tasks based on your cluster's available resources.
```
Comment (qiancai) [inline system-variables.md:1210]:
```suggestion
- This variable is used to set the concurrency within a single automatic statistics collection task. Before v8.4.0, the concurrency for each automatic statistics collection task is fixed at `1`. You can increase this concurrency to speed up statistics collection tasks based on your cluster's available resources.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**